iaf/testing/README.md

```diff
@@ -0,0 +1,85 @@
+# Test IAF Terraform Module
+
+## 1. Set up access to IBM Cloud
+
+If running this module from your local terminal, you need to set the credentials to access IBM Cloud.
+
+You can define the IBM Cloud credentials in the IBM provider block but it is recommended to pass them in as environment variables.
+
+Go [here](../../CREDENTIALS.md) for details.
+
+**NOTE**: These credentials are not required if running this Terraform code within an **IBM Cloud Schematics** workspace. They are automatically set from your account.
+
+## 2. Set OpenShift cluster environment variables
+
+This module test requires a running OpenShift cluster on IBM Cloud Classic with the basic MCM requirements. You need the cluster ID or name, and the resource group where it is running. These 2 parameters are passed to the testing code as environment variables. Execute this code:
+
+```bash
+export TF_VAR_cluster_id=<cluster_id>
+export TF_VAR_resource_group=<resource_group>
+```
+
+## 3. Set Cloud Pak Entitlement Key
+
+This module also requires an Entitlement Key. Obtain it [here](https://myibm.ibm.com/products-services/containerlibrary) and store it in the file `entitlement.key` in the root of this repository. If you use that filename, the file won't be published to GitHub if you accidentally push to GitHub. 
+
+## 4. Test
+
+### Using Terraform client
+
+Follow these instructions to test the Terraform Module manually
+
+Create the file `test.auto.tfvars` with the following input variables, these values are fake examples:
```

Comment:
This would seem clear if there was no mixing of instructions between steps 2 & 3 and step 4. Were 2 & 3 originally written for using the `Makefile`?

Reply:
Seems like I really didn't need to do step 2 (given the values in the `test.auto.tfvars`), but did have to do step 3

Reply:
> This would seem clear if there was no mixing of instructions between steps 2 & 3 and step 4. Were 2 & 3 originally written for using the `Makefile`?

Yes, I took out the Makefile steps.  They seemed redundant and weren't working.  Easier to just run TF commands.  Will remove this step.